.mvn/wrapper/maven-wrapper.properties

```diff
@@ -16,4 +16,4 @@
 # under the License.
 wrapperVersion=3.3.2
 distributionType=only-script
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.8/apache-maven-3.9.8-bin.zip
+distributionUrl=https://dist.apache.org/repos/dist/dev/maven/maven-3/3.9.9/binaries/apache-maven-3.9.9-bin.zip
```

Comment:
This is on Maven central now: https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.9/apache-maven-3.9.9-bin.zip